starlette/convertors.py

```diff
@@ -8,6 +8,9 @@
 class Convertor(typing.Generic[T]):
     regex: typing.ClassVar[str] = ""
 
+    def __init__(self, *args) -> typing.NoReturn:
```

Comment:
I guess `typing.NoReturn` is not really accurate it returns `None` right?

```suggestion
    def __init__(self, *args) -> None:
```

Reply:
Yes, You’re right I missed it

Reply:
Also, all the arguments are strings, since they are defined in the route, so

```
def __init__(self, *args: str) -> None:
```